src/App.tsx

```diff
@@ -328,26 +336,52 @@ function App() {
         }
     );
 
+    const switchToLiveSimulation = useCallback(
+        (layout: LayoutType) => {
+            if (trajectoryStatus === TrajectoryStatus.LOADED) {
+                simulariumController.clearFile();
+                setTrajectoryStatus(TrajectoryStatus.INITIAL);
+            }
+            if (
+                layout === LayoutType.LiveSimulation &&
+                simulariumController.getFile() !== LIVE_SIMULATION_NAME
+            ) {
+                // if we are on a live simulation page, and the file is not the live simulation
+                // load the live simulation
+                setTrajectoryName(LIVE_SIMULATION_NAME);
+            }
+        },
+        [simulariumController, trajectoryStatus]
+    );
+
+    // handle trajectory changes based on content changes
     useEffect(() => {
+        const currentPage = content[currentModule][page];
         const nextPage = content[currentModule][page + 1];
-        if (!nextPage) {
-            return;
-        }
-        const url = nextPage.trajectoryUrl;
-        if (trajectoryStatus === TrajectoryStatus.INITIAL && url) {
-            const changeTrajectory = async () => {
-                setIsPlaying(false);
-                setTrajectoryStatus(TrajectoryStatus.LOADING);
-                totalReset();
-
-                await fetch3DTrajectory(
-                    url,
-                    simulariumController,
-                    setTrajectoryPlotData
-                );
-                setTrajectoryStatus(TrajectoryStatus.LOADED);
-            };
-            changeTrajectory();
+        if (!nextPage?.trajectoryUrl && !currentPage.trajectoryUrl) {
+            // if there is not a 3D trajectory on this or the next page,
+            // clear any 3D trajectory and load a liveSim if necessary
+            switchToLiveSimulation(currentPage.layout);
```

Comment:
confused by nomenclature:  is 2D synonymous with Live, and 3D is coming from a url?

Reply:
yes it is in my case, but you're right I could make that clearer